src/main/java/mocacong/server/repository/MemberRepository.java

```diff
@@ -1,18 +1,21 @@
 package mocacong.server.repository;
 
-import java.util.Optional;
 import mocacong.server.domain.Member;
 import mocacong.server.domain.Platform;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.Query;
 
+import java.util.Optional;
+
 public interface MemberRepository extends JpaRepository<Member, Long> {
     Optional<Member> findByEmail(String email);
 
     Optional<Member> findByNickname(String nickname);
 
     Optional<Member> findByPlatformAndPlatformId(Platform platform, String platformId);
 
+    Optional<Member> findByEmailAndPlatform(String email, Platform platform);
+
```

Comment:
쓰이는 곳이 회원가입 시에 중복확인하는 경우밖에 없는 것 같아서, 아래 메서드도 괜찮을 것 같아요.

```suggestion
Boolean existsByPlatformAndEmail(Platform platform, String email);
```

Optional<Member> 반환형은 Member 속성의 모든 값을 가져오기 때문에 조금 더 부하가 클 듯 해요

Reply:
좋습니다~